### PR TITLE
Feature/add component test

### PR DESCRIPTION
**Pull request by jon-acker**
### What

Full component test for one feature: add new collection, using both ui and go BFF

### How to review

Run make test-component locally, watch browser, check test passes.

### CODE REVIEW

File: features/steps/florence_feature.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+	"testing"
+)
+
+type Chrome struct {
+	execAllocatorCanceller context.CancelFunc
+	ctxCanceller           context.CancelFunc
+	ctx                    context.Context
+}
+
+type FlorenceFeature struct {
```
[the remark this answers is not in the document]
Comment (jon-acker) [reply features/steps/florence_feature.go]: yup, agreed I will check for consistency.

File: features/creating_a_new_collection.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Feature: Creating a new Collection
+
+  As a Publisher etc.
```
[the remark this answers is not in the document]
Comment (jon-acker) [reply features/creating_a_new_collection.feature:3]: it's now changed from being uploading to creating a collection. hopefully it's up-to-date now;

**Review by cadmiumcat — DISMISSED** [review]
Do I need to do something in order to watch the browser? the tests happen very quickly, no browser pops up. The tests pass, although had to run `sudo make test-component`.

```
1 scenarios (1 passed)
5 steps (5 passed)
1.279279488s
PASS
coverage: 58.0% of statements in github.com/ONSdigital/florence/...
ok      github.com/ONSdigital/florence  1.783s
```

A couple of things:
- `README.md.gpg` - I assume that's a mistake?
- `go fmt ./...` 🙏

**Review by redhug1 — DISMISSED** [review]
LGTM
Code has been walked through with Jonathan and its execution demonstrated.

**Comment by jon-acker** [comment]
> Do I need to do something in order to watch the browser? the tests happen very quickly, no browser pops up. 

Sorry, just saw your comment, to watch the browser change the line `chromedp.Flag("headless", true),` in florence_component.go to `false`

Shouldn't have to use `sudo`, maybe a permissions issue?

Will run formatting command.

**Pull request closed without merging**